scene/2d/visibility_notifier_2d.cpp

```diff
@@ -252,10 +252,18 @@ void VisibilityEnabler2D::_notification(int p_what) {
 
 		_find_nodes(from);
 
-		if (enabler[ENABLER_PARENT_PHYSICS_PROCESS] && get_parent())
-			get_parent()->set_physics_process(false);
-		if (enabler[ENABLER_PARENT_PROCESS] && get_parent())
-			get_parent()->set_process(false);
+		/*We need to defer the call of set_process and set_physics_process, because	otherwise they are overwritten inside NOTIFICATION_READY.
+		We can't use call_deferred, because it happens after a physics frame.
+		The ready signal is emitted immediately after the NOTIFICATION_READY is finished, so it's perfect here.*/
```

Comment:
```suggestion
		// We need to defer the call of set_process and set_physics_process, otherwise they are
		// overwritten inside NOTIFICATION_READY. We can't use call_deferred, because it happens
		// after a physics frame. The ready signal is emitted immediately after the NOTIFICATION_READY
		// is finished, so it's perfect here.
```